settings.py

```diff
@@ -5,13 +5,4 @@
 # https://www.openmicroscopy.org/
 #   site/support/omero5.2/developers/Web/CreateApp.html
 
-
-def identity(x):
-    return x
-
-
-# import json
-CUSTOM_SETTINGS_MAPPINGS = {
-    "omero.web.figure.version":
-    ["OMERO_FIGURE_VERSION", "1.2.1", identity, None]
-}
+OMERO_FIGURE_VERSION = "1.2.1_dev"
```

Comment:
Is it worth reducing the number of versions to maintain in sync to one by loading the JSON file here e.g.:
```
import json
with open('package.json') as f:
    data = json.load(f)
OMERO_FIGURE_VERSION = data['version']
```